src/registrar/views/domain.py

```diff
@@ -377,7 +377,7 @@ def _make_invitation(self, email_address):
 
     def form_valid(self, form):
         """Add the specified user on this domain."""
-        requested_email = form.cleaned_data["email"]
+        requested_email = form.cleaned_data.get("email", "")
```

Comment:
why empty string not None? and then log if None

Reply:
Reverted this change, was not necessary